seatunnel-connectors-v2/connector-iceberg/src/main/java/org/apache/seatunnel/connectors/seatunnel/iceberg/config/CommonConfig.java

```diff
@@ -20,27 +20,62 @@
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkNotNull;
 
+import org.apache.seatunnel.api.configuration.Option;
+import org.apache.seatunnel.api.configuration.Options;
+
 import org.apache.seatunnel.shade.com.typesafe.config.Config;
 
 import lombok.Getter;
 import lombok.ToString;
 
 import java.io.Serializable;
+import java.util.List;
 
 @Getter
 @ToString
 public class CommonConfig implements Serializable {
     private static final long serialVersionUID = 239821141534421580L;
 
-    private static final String KEY_CATALOG_NAME = "catalog_name";
-    private static final String KEY_CATALOG_TYPE = "catalog_type";
-    private static final String KEY_NAMESPACE = "namespace";
-    private static final String KEY_TABLE = "table";
-    private static final String KEY_URI = "uri";
-    private static final String KEY_WAREHOUSE = "warehouse";
-    private static final String KEY_CASE_SENSITIVE = "case_sensitive";
+    public static final Option<String> KEY_CATALOG_NAME = Options.key("catalog_name")
+        .stringType()
+        .noDefaultValue()
+        .withDescription(" the iceberg catalog name");
+
+    public static final Option<String> KEY_CATALOG_TYPE = Options.key("catalog_type")
+        .stringType()
+        .noDefaultValue()
+        .withDescription(" the iceberg catalog type");
+
+    public static final Option<String> KEY_NAMESPACE = Options.key("namespace")
+        .stringType()
+        .noDefaultValue()
+        .withDescription(" the iceberg namespace");
+
+    public static final Option<String> KEY_TABLE = Options.key("table")
+        .stringType()
+        .noDefaultValue()
+        .withDescription(" the iceberg table");
+
+    public static final Option<String> KEY_URI = Options.key("uri")
+        .stringType()
+        .noDefaultValue()
+        .withDescription(" the iceberg server uri");
+
+    public static final Option<String> KEY_WAREHOUSE = Options.key("warehouse")
+        .stringType()
+        .noDefaultValue()
+        .withDescription(" the iceberg warehouse");
+
+    public static final Option<String> KEY_CASE_SENSITIVE = Options.key("case_sensitive")
+        .stringType()
+        .noDefaultValue()
```

Comment:
`case_sensitive` have a default value 
<img width="809" alt="image" src="https://user-images.githubusercontent.com/32193458/201452711-4e225d6c-1e14-40d1-a5e8-c092ef84dc93.png">

Reply:
Thank you.I will fix it.